### DISCUSSION TITLE
Unable to see Domain tab

### DISCUSSION THREAD

**Discussion by varmab** [swellstores/community#280] [Help] [answered]
I have a swell account on AppSumo plan for long time. I am unable to see Domain tab under storefront settings. How to fix custom domain for my store. I checked the documentation and other discussions in the forum. They all taking about Domain tab but it does not show in my admin panel. Can someone from Swell help me asap? My storefront id: jmFMWKgV

I only see Settings, Navigation and Developer tabs only. It does not show Domain any more. It showed few months back but not now.

Please help.

**Comment by logeshswell** [comment, marked as answer]
@varmab If you make your storefront (jmFMWKgV) **Primary**, then you could see `Domains` tab.

**Reply by josias-hosang** [reply]
Same problem here. I made my storefront (GEnwHsr7) **Primary** but still can not see the `Domains` tab.

**Reply by logeshswell** [reply]
@josias-hosang This is because you have a custom storefront for which you can apply the domain configurations from your hosting provider.

**Reply by josias-hosang** [reply]
Thanks @logeshswell. Isn't it possible to have a custom domain for checkout process (and admin dashboard): https://merz.swell.store/checkout ?

**Reply by logeshswell** [reply]
For checkout process? Yes
For admin dashboard? No

**Reply by ericingram** [reply]
@logeshswell I think we have some pending changes to checkout settings to configure domains. Aaron and Thomas were working on it. See if that's related to the issue here

**Reply by logeshswell** [reply]
Yes @ericingram, it also covers this use case.
